src/collection-view.js

```diff
@@ -295,7 +296,20 @@ Marionette.CollectionView = Marionette.AbstractView.extend({
       });
     }
 
-    return childView;
```

Comment:
This could be problematic actually. I remember us trying something like this earlier and running into bugs with identifying constructor functions. 

@MeoMix @samccone @jmeas recollections?